### DISCUSSION TITLE
How to copy file path to clipboard?

### DISCUSSION THREAD

**Discussion by tkossak** [leo-arch/clifm#197] [Q&A]
What's the quickest way to copy given file path to clipboard? Is there a command or shortcut for it? (I couldn't find any)

**Comment by leo-arch** [comment]
Hi @tkossak. There's a plugin (`clip`) that allows you to copy selected files to the system clipboard. Just run `clip --help` for more info.

**Comment by tkossak** [comment]
Is there a fast way of installing plugins? My clifm currently doesn't have `clip`, so should I just manually download it from github and place it in proper directory?

**Reply by leo-arch** [reply]
It should be there by default if running an up to date version. If not:

1. Download the [clip plugin](https://github.com/leo-arch/clifm/blob/master/plugins/clip.sh)
2. Copy it into `$HOME/.config/clifm/plugins/` (remember to make it executable)
3. In **clifm**, add an action to trigger the plugin:

```
actions edit
```
And then just add this line:

```
clip=clip.sh
```

Save and exit.
It should be there now and ready to use.

**Reply by tkossak** [reply]
`clip` plugin is not installed via aur package [clifm](https://aur.archlinux.org/packages/clifm), nor via `sudo make install` executed directly in cloned (and updated) clifm git repo. Ok I'll copy it manually.

When I run `actions` I currently (before making any changes to plugins) get only:
```
+      -> finder.sh
++     -> jumper.sh
-      -> fzfnav.sh
*      -> fzfsel.sh
**     -> fzfdesel.sh
//     -> rgfind.sh
_      -> fzcd.sh
bcp    -> batch_copy.sh
bmi    -> bm_import.sh
bn     -> batch_create.sh
cr     -> cprm.sh
da     -> disk_analyzer.sh
dr     -> dragondrop.sh
fdups  -> fdups.sh
gg     -> pager.sh
h      -> fzfhist.sh
i      -> img_viewer.sh
ih     -> ihelp.sh
kbgen  -> kbgen
kd     -> decrypt.sh
ke     -> encrypt.sh
ml     -> mime_list.sh
music  -> music_player.sh
ptot   -> pdf_viewer.sh
rrm    -> recur_rm.sh
update -> update.sh
vid    -> vid_viewer.sh
wall   -> wallpaper_setter.sh
```

**Reply by leo-arch** [reply]
Hi @tkossak. That's because you're still using old config files (in this case, the actions file), and **clifm** won't ever override user's config files. So, even if the plugin is there (and it should, since you're running an up to date version), you still need to add it to your actions list via `actions edit`, as described in my previous comment.
